server/util/redact/BUILD

```diff
@@ -9,8 +9,11 @@ go_library(
         "//proto:build_event_stream_go_proto",
         "//proto:command_line_go_proto",
         "//server/environment",
+        "//server/util/alert",
         "//server/util/git",
+        "//server/util/log",
         "@com_github_golang_protobuf//proto:go_default_library",
+        "@com_github_mattn_go_shellwords//:go-shellwords",
```

Comment:
we probably don't need any of these new imports anymore

Reply:
Fixed -- checkstyle noticed this too :)